app-backend/api/src/main/scala/user/Routes.scala

```diff
@@ -66,14 +65,6 @@ trait UserRoutes extends Authentication
     }
   }
 
-  def listUsers: Route = authenticateRootMember { user =>
-    withPagination { page =>
-      complete {
-        UserDao.query.page(page).transact(xa).unsafeToFuture
-      }
-    }
-  }
-
   def createUser: Route = authenticateRootMember { root =>
```

Comment:
Probably isn't a point in keeping this route around either - we never use it

Reply:
:gun: